README.md

```diff
@@ -1,4 +1,4 @@
-# Snowflake Snowpark Python API
+# Snowflake Snowpark Python and Snowpark pandas APIs
```

Comment:
```suggestion
# Snowflake Snowpark Python API
```

nit: again, it makes sense to me to leave the title as is, since Snowpark pandas is part of Snowpark python.

Reply:
Resolving this so we can continue the discussion on the first comment